tests/lib/thrift/ThriftTest/prj.conf

```diff
@@ -12,6 +12,7 @@ CONFIG_NETWORKING=y
 CONFIG_NET_SOCKETS=y
 CONFIG_NET_SOCKETPAIR=y
 CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_NEWLIB_LIBC_MAX_MAPPED_REGION_SIZE=4194304
```

Comment:
Since this is a config that is really only used for ZlibTransport, it should be listed in the extra_configs field in test case.yaml